app/src/main/java/com/kodeco/android/countryinfo/ui/components/ShimmeringLazyColumn.kt

```diff
@@ -0,0 +1,54 @@
+package com.kodeco.android.countryinfo.ui.components
+
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.facebook.shimmer.Shimmer
+import com.facebook.shimmer.ShimmerDrawable
```

Comment:
Note here the facebook library doesn't seem to be in the build.gradle.kts file. I had to remove these imports to get the app to run.  It doesn't look like the facebook library is utilized anymore anyway so removing the imports should be fine.

Reply:
Thanks for pointing it out. I forgot to remove it during final commit